next/utils/useAccount.ts

```diff
@@ -347,6 +347,7 @@ export default function useAccount(initStatus = AccountStatus.Idle) {
           })
           AWS.config.credentials = awsCredentials
 
+          accessToken = result.getAccessToken().getJwtToken()
```

Comment:
it's strange how accessToken is used here - or rather, it's not at all clear. 

It seems the only time it's used is in verify identity, that function is expecting it will have the correct instance coming from the closure, but that's relying on either the login function or the useEffect being called in the 'same render' .. ? Is this correct or am I missing something ?

If the above is the case, wouldn't it be preferred to rewrite accessToken to be a state variable ? `const [accessToken, setAccessToken] = useState('')`